### PR TITLE
Short array syntax

### PR DESCRIPTION
**Pull request by MathiasReker**

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

File: ps_sharebuttons.php [hunk 1]
```diff
@@ -95,43 +95,43 @@ public function getContent()
...
+                    ],
+                    [
                         'id' => Tools::strtolower($network).'_active_off',
                         'value' => 0,
-                        'label' => $this->trans('Disabled', array(), 'Admin.Global'),
-                    ),
+                        'label' => $this->trans('Disabled', [], 'Admin.Global'),
+                    ],
                 ),
             );
```
Comment (matks) [inline ps_sharebuttons.php]: Bad git merge

Comment (matks) [reply ps_sharebuttons.php]: Fixed in https://github.com/PrestaShop/ps_sharebuttons/pull/18/files#diff-3a8888c8085e5d6430bb741c68218053

**Comment by matks** [comment]
Fixed git conflicts.

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @MathiasReker

**Pull request merged**